app/domain/__init__.py

```
@@ -1,5 +1,6 @@
 from .accounts.model import Account
+from .currencies.model import Currency
 from .financial_institutions.model import FinancialInstitution
 from .user.model import User
 
-__all__ = ["User", "FinancialInstitution", "Account"]
+__all__ = ["User", "FinancialInstitution", "Account", "Currency"]
```

Ruff (0.15.1)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)